bundles/org.openhab.binding.lcn/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -163,7 +163,7 @@
 	</channel-type>
 
 	<channel-group-type id="rollershutterrelays">
-		<label>Roller Shutter on Relays</label>
+		<label>Roller Shutter (Relay)</label>
```

Comment:
Is using brackets a good idea? I get the feeling it may look ugly or cause a future bug as the label is used to create the item naming structure when you use the 'create equipment from thing' feature. In this case the item would be named:
thingName_RollerShutter(Relay)

Reply:
It looks like a good choice for the label to me, but if the item name is indeed derived from it, you might have a point.
@ghys Maybe you can comment here whether that's a problem?